aydin/nn/models/torch/test/test_torch_jinet.py

```diff
@@ -15,6 +15,14 @@ def test_forward_2D_jinet():
     assert result.dtype == input_array.dtype
 
 
+def test_forward_3D():
+    input_array = torch.zeros((1, 1, 128, 128, 128))
+    model3d = JINetModel(spacetime_ndim=3)
+    result = model3d(input_array)
+    assert result.shape == input_array.shape
+    assert result.dtype == input_array.dtype
```

Comment:
this test `test_forward_3D` is able to reproduce the error we are getting but I couldn't find why results of convolution shapes do not confirm with each other, can you give it a look @li-li-github when you have time?

Reply:
Sorry for the late reply. I managed to fetch this branch and tested it on my local and this test passed.

Reply:
okay, thank you @li-li-github !

Reply:
did you run on CPU @li-li-github ?